### PR TITLE
Feature/add bypass head

### PR DESCRIPTION
**Pull request by msaladin**
Hi there,

it seems that you forgot to include the advanced flag:
bypassHeadRequests

in the REST client. I added it, and use it already (works fine). Can you please accept this pull request and include this into future releases of the REST client.

Thank you & Best regards
Michael

### CODE REVIEW

**Comment by msaladin** [comment]
Any feedback on this?

Does it make sense or not? I thoughts were that all the fields which exist in the REST JSON response should be accessible using the official REST client?

Thanks for any feedback.

**Comment by eyalbe4** [comment]
@msaladin,
Thanks for this pull request and our apologies for the late response.
We'll soon look into your pull request and comment here.

**Comment by eyalbe4** [comment]
@msaladin,
Can you please add a test for the new repository attribute?
As a reference, you can loo at the *testClientTlsCertificate()* method.

File: services/src/main/java/org/jfrog/artifactory/client/model/impl/RemoteRepositoryBuilderImpl.java [hunk 1]
```diff
@@ -139,6 +162,15 @@ public boolean isEnableCookieManagement() {
         return enableCookieManagement;
     }
 
+    public RemoteRepositoryBuilder bypassHeadRequests(boolean theByPassBoolean) {
```
[the remark this answers is not in the document]
Comment (msaladin) [reply services/src/main/java/org/jfrog/artifactory/client/model/impl/RemoteRepositoryBuilderImpl.java]: done.

**Comment by msaladin** [comment]
Hi @eyalbe4,
I haved added two tests which run fine... I had some slight problems of repeating all the tests, it seems that the tests require a blank Artifactory instance to run against. But if this is the case, then the tests execute fine.
Let me know if that suits you. 
Best regards
Michael

**Pull request merged**